### PR TITLE
BEEFY: introduce offence report system

### PR DESCRIPTION
**Pull request by davxy**
https://github.com/paritytech/substrate/pull/13425 follow up

Please refer to the linked PR for details

(Note: this PR contains few modifications to the grandpa/babe docs that can be safely ignored)

---

polkadot companion: https://github.com/paritytech/polkadot/pull/6845

### CODE REVIEW

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/substrate/-/jobs/2499669

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/substrate/-/jobs/2499668

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/substrate/-/jobs/2499667

File: frame/beefy/src/equivocation.rs [hunk 1]
```diff
@@ -34,179 +34,205 @@
...
 			key_owner_proof,
 		};
 
-		match SubmitTransaction::<T, Call<T>>::submit_unsigned_transaction(call.into()) {
-			Ok(()) => log::info!(target: LOG_TARGET, "Submitted BEEFY equivocation report.",),
-			Err(e) =>
-				log::error!(target: LOG_TARGET, "Error submitting equivocation report: {:?}", e,),
+		let res = SubmitTransaction::<T, Call<T>>::submit_unsigned_transaction(call.into());
+		match res {
+			Ok(_) => info!(target: LOG_TARGET, "Submitted equivocation report.",),
```
Comment (melekes) [inline frame/beefy/src/equivocation.rs:162]:
```suggestion
			Ok(_) => info!(target: LOG_TARGET, "Submitted equivocation report."),
```

**Review by acatangiu — APPROVED** [review]

**Review by andresilva — APPROVED** [review]

**Comment by davxy** [comment]
bot merge

**Pull request merged**